specification/adp/resource-manager/readme.go.md

```diff
@@ -13,14 +13,12 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2020-07-01-preview
+  - tag: 2020-07-01-preview
+  - tag: 2021-02-01-preview
 ```
 
-### Tag: package-2020-07-01-preview and go
-
-These settings apply only when `--tag=package-2020-07-01-preview --go` is specified on the command line.
 Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
-```yaml $(tag) == 'package-2020-07-01-preview' && $(go)
-output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-07-01-preview/$(namespace)
+```yaml $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/$(tag)/$(namespace)
 ```
```

Comment:
We need to have two entries here - each corresponds to a tag in the `batch` above.
Also the `$(tag) == 'package-2020-07-01-preview'` is required, we cannot remove this.